includes/Core/Prompts/Dismissed_Prompts.php

```diff
@@ -41,12 +41,15 @@ class Dismissed_Prompts extends User_Setting {
 	public function add( $prompt, $expires_in_seconds = self::DISMISS_PROMPT_PERMANENTLY ) {
 		$prompts = $this->get();
 
+		$reference_date    = apply_filters( 'googlesitekit_reference_date', null );
```

Comment:
So far I believe we've only used this on the server side to pass along to the client to override the value used there. I'm not sure we should start using it to replace the current time on the backend. This requires more thought to do consistently across the codebase.